frontend/src/Wrapper.tsx

```diff
@@ -105,12 +103,15 @@ export default function Wrapper({ children }: WrapperProps): ReactElement {
         )}
         <Box
           component='main'
-          sx={{
-            backgroundColor: theme.palette.mode === 'light' ? theme.palette.grey[100] : theme.palette.grey[900],
+          sx={(theme) => ({
+            backgroundColor: theme.palette.grey[900],
```

Comment:
Maybe add a todo to use `theme.applyStyles` to set this for dark mode only in a future PR?